README.md

```diff
@@ -65,3 +66,25 @@ puma::nginxconfig {'myapp':
 ```
 
 This will create the necessary NGINX vhost and location configurations to serve a puma rails app.
+
+## Release Management
+
+The [`bump` gem](https://github.com/gregorym/bump) is used to manage version information in
+`CHANGELOG`, `metadata.json`, and Git tags. Currently this is a manual process. Insert a
+description of your changes at the top of the `CHANGELOG` file, leaving a blank line between
+your new description and the previous version entry. No header markup is necessary (`bump`
```

Comment:
Just a consideration, do we want to keep it this way, or would it be better to keep the #UNRELEASED header as per our "current" workflow.  (I'm not sure I have a preference either way, other than the #UNRELEASED header makes it clear what's going on in terms of "this being a work in progress or non-bumped change entry).

Reply:
The instructions here take advantage of `bump`'s built-in capabilities. (I'm guessing this was a feature added in recent versions.) The "`## Unreleased`" pattern relies on extra `sed` processing after running `bump`. Going forward, I suggest that code bases using the "`## Unreleased`" pattern should actually switch to simply using the `bump` option to add the version number to the CHANGELOG. I agree that the "`## Unreleased`" text does provide feedback, but in my tests bump simply ignores it and inserts the version number heading where we want it.